### PR TITLE
Add rem/2 to Nx.Defn.Kernel

### PR DESCRIPTION
**Pull request by dannote**

### CODE REVIEW

File: nx/lib/nx/defn/kernel.ex [hunk 1]
```diff
@@ -403,6 +403,27 @@ defmodule Nx.Defn.Kernel do
...
+  It delegates to `Nx.remainder/2` (supports broadcasting).
+
+  ## Examples
+
+      defn divides_by_5?(a) do
+        rem(a, 5)
+        |> Nx.any?
+        |> Nx.equal(Nx.tensor(1))
+      end
+
```
Comment (polvalente) [inline nx/lib/nx/defn/kernel.ex:419]:
```suggestion
```

Comment (polvalente) [reply nx/lib/nx/defn/kernel.ex:419]: @josevalim I just noticed that the other docs in this file tend to end on a blank line. Does this make any difference?

File: nx/lib/nx/defn/kernel.ex [hunk 2]
```diff
@@ -403,6 +403,26 @@ defmodule Nx.Defn.Kernel do
...
+  """
+
```
Comment (josevalim) [inline nx/lib/nx/defn/kernel.ex:420]:
```suggestion
```

File: nx/lib/nx/defn/kernel.ex (hunk 1 above)
Comment (josevalim) [reply nx/lib/nx/defn/kernel.ex:418]: No, it does not matter, but that's the "convention" we follow in Elixir (and apparently Nx).

**Review by josevalim — APPROVED** [review]

File: nx/lib/nx/defn/kernel.ex (hunk 1 above)
Comment (polvalente) [reply nx/lib/nx/defn/kernel.ex:418]: I see!

**Comment by josevalim** [comment]
:green_heart: :blue_heart: :purple_heart: :yellow_heart: :heart:

**Pull request merged**